platform/lib/schema/__tests__/__snapshots__/MapOfSetting.test.ts.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`fails when not receiving expected key type 1`] = `"[name]: expected value of type [number] but got [string]"`;
```

Comment:
These errors aren't fantastic at giving you context yet (e.g. this doesn't separate between if the key failed or the value failed), but I think that's a separate PR cleaning it up across the Setting stuff.